AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.spec.ts

```diff
@@ -19,11 +19,11 @@ import { Logger } from '../../services/logger';
 import { SearchService } from '../../services/search.service';
 import { VideoHearingsService } from '../../services/video-hearings.service';
 import { RemovePopupStubComponent } from '../../testing/stubs/remove-popup-stub';
-import { ParticipantsListComponent } from '../participants-list/participants-list.component';
 import { SearchEmailComponent } from '../search-email/search-email.component';
 import { ParticipantService } from '../services/participant.service';
 import { AddParticipantComponent } from './add-participant.component';
 import { HearingRoleModel } from '../../common/model/hearing-role.model';
+import { ParticipantListComponent } from '../participant';
```

Comment:
other component file names end in .component
should this be called `participant-list` (`-component`) as its not just a participant?

Reply:
`participant` is just the folder - I've added an index.ts to shorten the paths a bit

Reply:
that folder contains both the `list` and the `item`